specutils/analysis/flux.py

```diff
@@ -52,7 +52,7 @@ def equivalent_width(spectrum, continuum=1, regions=None):
     spectrum : Spectrum1D
         The spectrum object overwhich the equivalent width will be calculated.
 
-    regions: `~specutils.spectra.spectralregion.SpectralRegion` or list of `~specutils.spectra.spectralregion.SpectralRegion`
+    regions: `~specutils.spectra.spectral_region.SpectralRegion` or list of `~specutils.spectra.spectral_region.SpectralRegion`
```

Comment:
I would suggest to drop the `spectral_region` part altogether as `SpectralRegion` is available from the `specutils.spectra` namespace.

On another note I wonder why this didn't raise a sphinx WARNING and thus failing the docs build CI job?